lib/stat_tracker.rb

```diff
@@ -25,27 +25,24 @@ def team_list
     @teams.each_with_object(Hash.new) { |team, team_list| team_list[team.team_id] = team.team_name}
   end
   
-  def best_offense
+  def average_goals_per_team_id
```

Comment:
Looks good to break it out into this helper method for better SRP. I'm sure we could refactor further and simplify if need be, but we can focus on moving on, and refactor as our last steps